docs(install-dependencies): add stop the openresty after installed (#4521)
apache/apisix: docs/en/latest/install-dependencies.md

- WARNING: If you are using OpenResty which is older than `1.17.8`, please installing openresty-openss-devel instead of openresty-openssl111-devel.

> - OpenResty is a dependency of APISIX, if it is the first time to deploy APISIX and you don't need to use OpenResty to deploy other services, you can stop/disable OpenResty after OpenResty installation is finished, this will not affect the normal work of APISIX, please operate carefully according to your service. For example in Ubuntu: `systemctl stop openresty && systemctl disable openresty`.

Reviewer: ```suggestion
- OpenResty is a dependency of APISIX, if it is the first time to deploy APISIX and you don't need to use OpenResty to deploy other services, you can stop and disable OpenResty after OpenResty installation is finished, this will not affect the normal work of APISIX, please operate carefully according to your service. For example in Ubuntu: `systemctl stop openresty && systemctl disable openresty`.
```

> - OpenResty is a dependency of APISIX, if it is the first time to deploy APISIX and you don't need to use OpenResty to deploy other services, you can stop and disable OpenResty after OpenResty installation is finished, this will not affect the normal work of APISIX, please operate carefully according to your service. For example in Ubuntu: `systemctl stop openresty && systemctl disable openresty`.

Reviewer 2: ```suggestion
- OpenResty is a dependency of APISIX. If it is your first time to deploy APISIX and you don't need to use OpenResty to deploy other services, you can stop and disable OpenResty after installation since it will not affect the normal work of APISIX. Please operate carefully according to your service. For example in Ubuntu: `systemctl stop openresty && systemctl disable openresty`.
```
